protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -175,6 +175,8 @@ message Operation {
   // extra functionality that is not covered by the standard OpenAPI Specification.
   // See: https://swagger.io/docs/specification/2-0/swagger-extensions/
   map<string, google.protobuf.Value> extensions = 13;
+
+  map<string, Header> headers = 14;
```

Comment:
The fields in this msaage map to the fields in https://swagger.io/specification/v2/#operation-object. As far as I can tell, headers does not belong here, but in the parameter object: https://swagger.io/specification/v2/#parameter-object. Naturally, we don't actually allow the user to specify the `parameter` object, since it's entirely inferred from the request parameters in the protobuf file. Maybe you could define a very shallow parameter type and we can allow the user to only specify headers? Still not sure it's a good idea though.

Reply:
Okay, I will try that.